### PR TITLE
Ga 48 create table profile refresh queue

### PR DESCRIPTION
**Pull request by patelila**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/GA-48

### Change description ###

PRM : Create table profile_refresh_queue

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/orgrolemapping/data/ProfileRefreshQueueTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.mock;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ProfileRefreshQueueTest {
+
+    private final ProfileRefreshQueueRepository profileRefreshQueueRepository
```
Comment (sanjayparekh-hmcts) [inline src/test/java/uk/gov/hmcts/reform/orgrolemapping/data/ProfileRefreshQueueTest.java:15]: is this a test for the PRQ object or repository?
you seem to be mixing here?

**Review by sanjayparekh-hmcts — CHANGES_REQUESTED** [review]

File: src/test/java/uk/gov/hmcts/reform/orgrolemapping/data/ProfileRefreshQueueTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.mock;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ProfileRefreshQueueTest {
```
Comment (abul-syed) [inline src/test/java/uk/gov/hmcts/reform/orgrolemapping/data/ProfileRefreshQueueTest.java:13]:
```suggestion
class ProfileRefreshQueueTest {
```

code smell

**Review by helenbirdhmcts — CHANGES_REQUESTED** [review]

**Review by helenbirdhmcts — APPROVED** [review]
lgtm

**Review by sanjayparekh-hmcts — DISMISSED** [review]

**Review by abul-syed — DISMISSED** [review]

**Review by helenbirdhmcts — APPROVED** [review]

**Review by abul-syed — APPROVED** [review]

**Review by sanjayparekh-hmcts — APPROVED** [review]

**Review by poojithanag — APPROVED** [review]
QA Approved

**Pull request merged**